src/util/Parameters.h

```diff
@@ -99,6 +122,21 @@ struct Parameter : public ParameterBase {
   }
 };
 
+// Concept for `Parameter`.
+namespace detail::parameterConceptImpl {
+template <typename T>
+struct ParameterConceptImpl : std::false_type {};
+
+template <ParameterValueType Type, ParameterFromStringType<Type> FromString,
+          ParameterToStringType<Type> ToString, ParameterName Name>
+struct ParameterConceptImpl<Parameter<Type, FromString, ToString, Name>>
+    : std::true_type {};
+}  // namespace detail::parameterConceptImpl
+
+template <typename T>
+concept isParameter =
```

Comment:
I think we use Uppercase for concepts, but I am not sure.

Reply:
Done in [d037cae](https://github.com/ad-freiburg/qlever/pull/1067/commits/d037cae0894814353395723d0391cf3d5803948c).